transport/ca/cfssl_provider.go

```diff
@@ -206,13 +206,17 @@ func (cap *CFSSL) SignCSR(csrPEM []byte) (cert []byte, err error) {
 		return nil, err
 	}
 
-	hosts := make([]string, len(csr.DNSNames), len(csr.DNSNames)+len(csr.IPAddresses))
+	hosts := make([]string, len(csr.DNSNames), len(csr.DNSNames)+len(csr.IPAddresses)+len(csr.URIs))
```

Comment:
I know this was existing code, but there's no sense in making the array the right length if it's being appended to, see https://goplay.space/#6CL-nQJOxsS

```suggestion
	hosts := make([]string, 0)
```

Reply:
Thanks!